docs/how-tos/hello-world-module.md

```diff
@@ -0,0 +1,508 @@
+   For example, `jessamy:hello-world:hello-sensor`.
+
+   For type, enter `sensor`.
+
+   For name, you can use the automatic `sensor-1`.
+
+10. Save the config, then click **TEST** to see a random number generated every second.
+
+    ![The sensor card test section open.](/how-tos/hello-sensor.png)
+
+{{< /expand >}}
+
+## Package and upload the module
+
+You have a working local module.
+You can make it available to deploy on more machines by packaging it and uploading it to the [Viam registry](https://app.viam.com/registry).
+
+The hello world module you created is for learning purposes, not to provide any meaningful utility, so we recommend making it available only to machines within your {{< glossary_tooltip term_id="organization" text="organization" >}} instead of making it publicly available.
+
+{{< expand "Click to see what you would do differently if this wasn't just a hello world module" >}}
+
+1. Create a GitHub repo with all the source code for your module.
+   Add the link to that repo as the `url` in the <file>meta.json</file> file.
+1. Create a README to document what your module does and how to configure it.
+1. If you wanted to share the module outside of your organization, you'd set `"visibility": "public"` in the <file>meta.json</file> file.
+
+{{< /expand >}}
+
+To package and upload your module and make it available to configure on machines in your organization:
+
+1. Package the module as an archive, run the following command from inside the <file>hello-world</file> directory:
+
+    ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+    tar -czf module.tar.gz run.sh setup.sh requirements.txt src
+    ```
+
+    This creates a tarball called <file>module.tar.gz</file>.
+
+1. Run the `viam module upload` CLI command to upload the module to the Viam registry:
+
+    ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+    viam module upload --version 1.0.0 --platform any module.tar.gz
+    ```
+
+1. Now, if you look at the [Viam Registry page](https://app.viam.com/registry) while logged into your account, you'll be able to find your private module listed.
+   You can configure the hello-sensor and hello-camera on your machines just as you would configure other components and services; there's no more need for local module configuration.
+
+    ![The create a component menu open, searching for hello. The hello-camera and hello-sensor components are shown in the search results.](/how-tos/hello-config.png)
+
+For more information about uploading modules, see [Upload a module](/how-tos/upload-module/).
+
+## Next steps
+
+For an example module that gets weather data from an online source, see the example code in [Create a sensor module with Python](/how-tos/sensor-module/).
```

Comment:
Bec
```suggestion
For an example module that creates different kinds of sensor models, for example to get weather data from an online source, see the example code in [Create a sensor module with Python](/how-tos/sensor-module/).
```

Reply:
I don't think it's a categorically different kind of sensor model?

Reply:
- The guide shows multiple models rather than just one, so this feels weird.
- My suggestion added the word example three times and made the sentence very long, I think this is better

```suggestion
For a guide that walks you through creating different sensor models, for example to get weather data from an online source, see [Create a sensor module with Python](/how-tos/sensor-module/).
```